index.html

```diff
@@ -64,8 +64,31 @@
     <h2>
       Introduction
     </h2>
+    <p>
+      This document defines an API enabling a website to request presentation of a <dfn data-abbr>verifiable digital credential</dfn> (VDC).
+    </p>
+    <p>
+      The API design is agnostic to both credential presentation exchange protocols and credential formats. However, this document includes a [[[#protocol-registry]]] to ensure interoperability.
+    </p>
+    <p>
+      The API is designed to support the following goals:
```

Comment:
Perhaps also worth adding some other key things we know make this attractive over custom schemes?
 - Enables platform-provided credential selection UX for when multiple wallet applications have credentials which match a request
 - Enables platforms to potentially provide secure cross-device requests with proximity checks